flowey/flowey_cli/src/pipeline_resolver/ado_yaml.rs

```diff
@@ -397,6 +406,16 @@ EOF
             })
         }
 
+        ado_steps.push({
```

Comment:
now that this has moved down here, I think you should also shift some code around to move the buildup of `flowey_bootstrap_bash` to be a bit closer to where its emitted.

i.e: make it easier to "straight-line read" for someone trying to grok this code in the future

Reply:
ditto for the GitHub backend